modules/openapi-generator/src/main/resources/r/modelGeneric.mustache

```diff
@@ -238,7 +238,11 @@
           self$`{{name}}`
           {{/isPrimitiveType}}
           {{^isPrimitiveType}}
-          self$`{{name}}`$toJSON()
+          if (length(names(self$`{{name}}`$toJSON())) == 0L && is.character(jsonlite::fromJSON(self$`{{name}}`$toJSON()))) {
```

Comment:
i saw you using is.character check

Does it also cover numeric values such as 13, 5.6  ?

Reply:
Currently it does not, as I intended. I only want to catch enum classes (as generated by `modelEnum.mustache`) and did not see how anything other than character would make much sense. If you want to include numeric values, I could check for `jsonlite::fromJSON(self$`{{name}}`$toJSON()) %in$ c("character", "numeric", "integer", "logical", "complex")` instead of using `is.character`.